.github/workflows/self-test.yml

```diff
@@ -0,0 +1,59 @@
+# Use this file during development to test changes to the action.
```

Comment:
We should use this syntax here: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-action-in-the-same-repository-as-the-workflow That way the test will always run on the correct code without additional user work.

Reply:
Thank you! I was googling for this but could not find.